### DISCUSSION TITLE
怎么运行项目呀

### DISCUSSION THREAD

**Discussion by Bubbfish** [jd-opensource/drip-form#40] [Q&A] [answered]
如题，，求教，我把项目克隆下来后，npm i ，npm run start:dripform之后不行

**Comment by mengshang918** [comment, marked as answer]
因为用的yarn workspace + lerna管理的，所以要使用yarn安装依赖
1. yarn
2. yarn build:prod
3. yarn start:generator

如果需要watch变更，使用yarn watch:ts
start:dripform案例已经删除了，这边更新下；这几天会补充下CONTRIBUTIN.md文档

**Reply by Bubbfish** [reply]
谢谢呀！

**Reply by Bubbfish** [reply]
我发现个问题
按照你的步骤，我重新clone项目之后，在yarn start:generator的时候会报一个错
 src/App/index.jsx:2:30: note: The plugin "vite:dep-scan" was triggered by this import
        2 │ import DripFormGenerator from '@jdfed/form-generator'

我版本是 yarn 1.22.17 node  14.18.1 npm 6.14.15 ；

我这样解决的：
我直接进入example，在进入form-generator,然后 npm安装，就安装了上面那个错误的包，在运行命令就成功了。
